django_project/ta_system/forms.py

```diff
@@ -54,7 +54,7 @@ class Meta:
 
 
 class UserUpdateForm(forms.ModelForm):
-    username = forms.EmailField(max_length=30, label="BC Email",
+    username = forms.EmailField(max_length=30, label="BC Email", disabled=True,
```

Comment:
You can remove these changes, because they are also included in a PR I'm either about to or have just merged. Also note that because we are making these form fields disabled, it would make more sense to remove other attributes that would no longer be relevant, which is everything except the label.